src/fail.js

```diff
@@ -0,0 +1,38 @@
+var _curry2 = require('./internal/_curry2');
+var _isFunction = require('./internal/_isFunction');
+var toString = require('./toString');
+
+
+/**
+ * Returns the result of applying the onFailure function to the value inside
+ * a failed promise. This is useful for handling rejected promises
+ * inside function compositions.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (b -> c) -> (Promise a b) -> (Promise c d)
+ * @sig (b -> (Promise c d)) -> (Promise a b) -> (Promise c d)
+ * @param {Function} onFailure The function to apply.  Can return a value or a promise.
```

Comment:
> `apply.  Can`

extra space?

Reply:
Heh. I've always put 2 spaces after a period... I'll fix it.

Reply:
Yeah, me too.  I know it's not the most common choice nowadays, but it's so hard to break.

...

> `me too.  I know`
> `......   ^--- see?`